Khan/perseus
# [Locked Figure Labels] Locked points: Add the locked point labels UI to the editor

## Summary:
- Add the ability to add/edit/delete locked point labels from locked point settings
- Pass down flags
- Add tests

Issue: https://khanacademy.atlassian.net/browse/LEMS-2346

## Test plan:
- Go to http://localhost:6006/?path=/story/perseuseditor-widgets-interactive-graph--mafs-with-locked-point-labels-flag
- Go to locked figures
- Open the locked point settings
- confirm that the label settings are there
- Play around with the label settings and confirm that it moves
  and updates with the point

<img width="376" alt="image" src="https://github.com/user-attachments/assets/eef73bb2-d6d1-4757-8d0b-54a4853cfe5f">
